tests/bluetooth/bsim_bt/bsim_test_adv_resume/src/central.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_bt_utils.h"
+#include "zephyr/bluetooth/addr.h"
+#include "zephyr/bluetooth/conn.h"
+
+#include <stdint.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+
+void adv_new_ident(void)
+{
+	/* create a new identity:
+	 * the stack disallows connecting to the same peer twice
+	 */
+	int id_a = bt_id_create(NULL, NULL);
```

Comment:
Why do we need to connect to the same device in the first place? It should be trivial to add more devices to the test.

Reply:
It is easier to coordinate two devices instead of three. We're only interested in making 3 connections, we don't really need 3 devices to reproduce a logic bug. Ideally, this would've been an unit test.

Reply:
If it is logic behavior, why we do need 2 connections in the first place? 

IMO it would be a better solution to do it "properly" with 2 actual connections to devices, rather than multiple connections to the same device, unless it is specifically needed to have multiple connections to the same device. I assume that whatever we are testing here are to verify that it works against multiple devices, right? The test case is better off simulating the actual scenario.

Reply:
You're right, I will edit it accordingly.